src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -313,21 +323,155 @@ public static void main(final String[] args) {
         return 0;
     }
 
-    protected static GenotypesContext fixGenotypes(final GenotypesContext originals, final Map<Allele, Allele> reverseComplementAlleleMap) {
+    protected static Locatable getFlippedLocation(final VariantContext source, final LiftOver liftOver) {
+        Interval originalLocus = new Interval(source.getContig(), source.getStart() + 1, source.getEnd() + 1);
+        return liftOver.liftOver(originalLocus);
+    }
+
+    /**
+     * @param source            original variant context
+     * @param liftOver          the LiftOver object to use for flipping
+     * @param referenceSequence the reference sequence of the target
+     * @return a flipped variant-context.
+     */
+    protected static VariantContext flipIndel(final VariantContext source, final LiftOver liftOver, final ReferenceSequence referenceSequence) {
+        if (!source.isBiallelic()) return null;  //only supporting biallelic indels, for now.
+
+        final Locatable target = getFlippedLocation(source, liftOver);
+        if (target == null) return null;
+
+        final Map<Allele, Allele> reverseComplementAlleleMap = new HashMap<>(10);
+
+        reverseComplementAlleleMap.clear();
+        final List<Allele> alleles = new ArrayList<>();
+
+        for (final Allele oldAllele : source.getAlleles()) {
+            // target.getStart is 1-based, reference bases are 0-based
+            final String alleleWithoutHead = oldAllele.getBaseString().substring(1, oldAllele.length());
+            final char refBase = (char) referenceSequence.getBases()[target.getStart() - 1];
+            final Allele fixedAllele = Allele.create(refBase + SequenceUtil.reverseComplement(alleleWithoutHead), oldAllele.isReference());
+            alleles.add(fixedAllele);
+            reverseComplementAlleleMap.put(oldAllele, fixedAllele);
+        }
+
+        VariantContextBuilder builder = new VariantContextBuilder(source.getSource(),
+                target.getContig(),
+                target.getStart(),
+                target.getEnd(),
+                alleles);
+
+        builder.id(source.getID());
+        builder.attributes(source.getAttributes());
+
+        builder.genotypes(fixGenotypes(source.getGenotypes(), reverseComplementAlleleMap));
+        builder.filters(source.getFilters());
+        builder.log10PError(source.getLog10PError());
+
+        return leftAlignVariant(builder.make(), referenceSequence);
```

Comment:
But what if the reference has changed and now the indel can be left aligned even more on the new one?  I guess this would be rare enough that we can ignore it.